WordPress/src/main/res/layout/login_2fa_screen.xml

```diff
@@ -64,10 +64,12 @@
                     style="@style/Base.TextAppearance.AppCompat.Caption"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:minHeight="48dp"
```

Comment:
We should add a `dimen` resource for this - I'm sure we'll use it again. Maybe `min_touch_target_sz`?